src/session.ts

```diff
@@ -412,12 +456,15 @@ export async function getListHtml(webview: Webview, file: string): Promise<strin
   <link href="${String(webview.asWebviewUri(Uri.file(path.join(resDir, 'jquery.json-viewer.css'))))}" rel="stylesheet">
   <style type="text/css">
     body {
-        color: black;
-        background-color: white;
+        color: var(--vscode-editor-foreground);;
+        background-color: var(--vscode-editor-background);;
     }
     pre#json-renderer {
       border: 1px solid #aaa;
```

Comment:
Adding the following seems to make the list view nicer for other themes:

```css
pre#json-renderer {
      border: 1px solid var(--vscode-editorLineNumber-foreground);
}

ul.json-dict, ol.json-array {
        border-left: 1px dotted var(--vscode-editorLineNumber-foreground);
}
```

E.g.:

![image](https://user-images.githubusercontent.com/60372411/125916524-76fced75-db84-4a0a-ad99-5cc149f6f42a.png)

![image](https://user-images.githubusercontent.com/60372411/125916486-e83bae27-b3a2-481a-bb03-abfa174c6c51.png)

![image](https://user-images.githubusercontent.com/60372411/125916568-60821964-dce5-42a2-9bba-6550b979f19b.png)

Reply:
Definitely! Thanks!